rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -1810,6 +1810,94 @@ public void rskTxWaitingForSignature_uses_updateCollection_rskTxHash_before_rski
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test
+    public void rskTxWaitingForSignature_preRSKIP326_emitNewPegoutConfirmedEvent() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP326)).thenReturn(true);
+
+        BridgeConstants spiedBridgeConstants = spy(BridgeRegTestConstants.getInstance());
+        doReturn(1).when(spiedBridgeConstants).getRsk2BtcMinimumAcceptableConfirmations();
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+        BtcTransaction btcTx = mock(BtcTransaction.class);
+        Set<ReleaseTransactionSet.Entry> set = new HashSet<>();
```

Comment:
```suggestion
        Set<ReleaseTransactionSet.Entry> releaseTransactionSetEntries = new HashSet<>();
```
Let's try to use meaningful variable names